app/cafe.py

```diff
@@ -0,0 +1,35 @@
+import datetime
+
+from app.errors import (
+    VaccineError,
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(
+            self,
+            visitor: dict
+    ) -> str | VaccineError | NotWearingMaskError:

```

Comment:
this function can raise errors and can not return errors 
```suggestion
    ) -> str:

```

Reply:
understood. Thx a lot !